eo-runtime/src/main/java/org/eolang/PhDefault.java

```diff
@@ -117,8 +105,9 @@ public boolean equals(final Object obj) {
     }
 
     @Override
+    @SuppressWarnings("PMD.UselessOverridingMethod")
```

Comment:
@maxonfjvipon maybe you can just delete the method?)

Reply:
@yegor256 PMD wants us to have either both methods `equals` and `hashCode` or nothing. If we delete them both - object comparison in some test stops working, for example in `org.eolang.PhDefaultTest#doesNotChangeRhoAfterDirectKidCopying`

Reply:
@maxonfjvipon implement it like this and remove the warning:

```
return super.hashCode() + 1;
```